src/language-js/printer-estree.js

```diff
@@ -3913,7 +3913,11 @@ function printFunctionParams(path, print, options, expandArg, printTypeParams) {
     parts.push("this: ");
     parts.push(path.call(print, "this"));
     if (printed.length > 0 || fun.rest) {
-      parts.push(",", line);
+      if (isNextLineEmpty(options.originalText, fun.this, options.locEnd)) {
```

Comment:
I think this is really bad idea

Reply:
I just took this from how parameters are handled above. What specifically about it do you think is bad?

Reply:
We should avoid to using original source code

Reply:
In that case what would you suggest instead?

Reply:
It's the right thing to do. Prettier preserves blank lines between parameters.

Reply:
https://github.com/prettier/prettier/pull/5836